Lecture-6/Task1/l6task1/Application.java

```diff
@@ -0,0 +1,139 @@
+package l6task1;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Application {
+
+    private static String[] coffeeType = {"bean", "ground", "instant"};
+    private static String[] coffeeSort = {"arabica", "robusta"};
+    private static int[] coffeePrice = {10, 20, 30};
+    private static String[] packType = {"can", "bag"};
+    private static int[] packCapacity = {50, 100, 200, 300, 500};
+    private static int[] packValue = {5, 10, 20, 30, 50};
+    private static int[] packPrice = {10, 20, 30, 40, 50};
+    private static String[] brands = {"Tchibo", "Pele", "Nescafe", "Jacobs", "Maxwell"};
+    private static Van van;
+
+    public static String getRandomProductBrand() {
+        return brands[(int) (Math.random() * 5)];
+    }
+
+    public static List<Pack> createPacks() {
+        List<Pack> packs = new ArrayList<Pack>();
+        for (int i = 0; i < packCapacity.length; i++) {
+            Pack canPack = new Pack(packType[0], packValue[i], packPrice[i], packCapacity[i]);
+            packs.add(canPack);
+            Pack bagPack = new Pack(packType[1], packValue[i], 2 * packPrice[i], packCapacity[i]);
+            packs.add(bagPack);
+        }
+        return packs;
+    }
+
+    public static List<Coffee> createCoffee() {
+        List<Coffee> coffees = new ArrayList<Coffee>();
+        for (int i = 0; i < coffeeType.length; i++) {
+            Coffee coffeeArabica = new Coffee(coffeeType[i], coffeeSort[0], coffeePrice[i]);
+            Coffee coffeeRobusta = new Coffee(coffeeType[i], coffeeSort[1], 2 * coffeePrice[i]);
```

Comment:
inline `coffeeRobusta ` and `coffeeArabica `

Reply:
done